applications/consdb/values-base.yaml

```
@@ -0,0 +1,13 @@
+environment:
+  KAFKA_REST: "https://base-lsp.lsst.codes/sasquatch-rest-proxy"
+  KAFKA_BOOTSTRAP: "sasquatch-base-kafka-bootstrap.lsst.codes"
+  vaultUrl: "https://vault.lsst.codes"
```

There is a new Vault URL. Is your app directly getting secrets from Vault rather than using Vault Secrets Operator? If you don't need this, I'd suggest deleting it.

```suggestion
  vaultUrl: "https://vault.lsst.cloud"
```

---

i believe i can remove this .. i use the operator